applications/GeoMechanicsApplication/custom_conditions/U_Pw_face_load_interface_condition.cpp

```diff
@@ -75,14 +75,14 @@ void UPwFaceLoadInterfaceCondition<TDim,TNumNodes>::
 {        
     //Previous definitions
     const GeometryType& Geom = this->GetGeometry();
-    const GeometryType::IntegrationPointsArrayType& integration_points = Geom.IntegrationPoints( mThisIntegrationMethod );
-    const unsigned int NumGPoints = integration_points.size();
+    const GeometryType::IntegrationPointsArrayType& IntegrationPoints = Geom.IntegrationPoints( mThisIntegrationMethod );
```

Comment:
```suggestion
    const GeometryType::IntegrationPointsArrayType& r_integration_points = Geom.IntegrationPoints( mThisIntegrationMethod );
```

this would be the correct one according to the style guide

Reply:
Thanks @philbucher for the suggestion. I will consider it in my next PR.